### PR TITLE
chore: update alias across code base and set in golint

### PR DESCRIPTION
**Pull request by zdtsw**
## Merge criteria

- [ ] You have read the [contributors guide](https://github.com/opendatahub-io/opendatahub-operator/blob/incubation/CONTRIBUTING.md).
- [ ] Commit messages are meaningful - have a clear and concise summary and detailed explanation of what was changed and why.
- [ ] Pull Request contains a description of the solution, a link to the JIRA issue, and to any dependent or related Pull Request.
- [ ] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [ ] The developer has manually tested the changes and verified that the changes work

### CODE REVIEW

File: .golangci.yml [hunk 1]
```diff
@@ -45,6 +45,10 @@ linters-settings:
         alias: $1$2
       - pkg: github.com/openshift/api/(\w+)/(v[\w\d]+)
         alias: $1$2
+      - pkg: github.com/opendatahub-io/opendatahub-operator/v2/controllers/datasciencecluster
+        alias: dsccontr
+      - pkg: github.com/opendatahub-io/opendatahub-operator/v2/controllers/dscinitialization
+        alias: dscicontr
```
Comment (bartoszmajsak) [inline .golangci.yml:51]:
```suggestion
      - pkg: github.com/opendatahub-io/opendatahub-operator/v2/controllers/datasciencecluster
        alias: dscctrl
      - pkg: github.com/opendatahub-io/opendatahub-operator/v2/controllers/dscinitialization
        alias: dscictrl
```

File: main.go [hunk 2]
```diff
@@ -53,7 +53,7 @@ import (
 	dsciv1 "github.com/opendatahub-io/opendatahub-operator/v2/apis/dscinitialization/v1"
 	featurev1 "github.com/opendatahub-io/opendatahub-operator/v2/apis/features/v1"
 	"github.com/opendatahub-io/opendatahub-operator/v2/controllers/certconfigmapgenerator"
-	datascienceclustercontrollers "github.com/opendatahub-io/opendatahub-operator/v2/controllers/datasciencecluster"
+	dsccontr "github.com/opendatahub-io/opendatahub-operator/v2/controllers/datasciencecluster"
```
Comment (bartoszmajsak) [inline main.go:56]:
```suggestion
	dscctrl "github.com/opendatahub-io/opendatahub-operator/v2/controllers/datasciencecluster"
```

File: main.go [hunk 3]
```diff
@@ -160,11 +160,11 @@ func main() { //nolint:funlen
 		os.Exit(1)
 	}
 
-	if err = (&datascienceclustercontrollers.DataScienceClusterReconciler{
+	if err = (&dsccontr.DataScienceClusterReconciler{
```
Comment (bartoszmajsak) [inline main.go:163]:
```suggestion
	if err = (&dscctrl.DataScienceClusterReconciler{
```

File: main.go [hunk 4]
```diff
@@ -160,11 +160,11 @@ func main() { //nolint:funlen
...
 		Client: mgr.GetClient(),
 		Scheme: mgr.GetScheme(),
 		Log:    logger.LogWithLevel(ctrl.Log.WithName(operatorName).WithName("controllers").WithName("DataScienceCluster"), logmode),
-		DataScienceCluster: &datascienceclustercontrollers.DataScienceClusterConfig{
+		DataScienceCluster: &dsccontr.DataScienceClusterConfig{
```
Comment (bartoszmajsak) [inline main.go:167]:
```suggestion
		DataScienceCluster: &dscctrl.DataScienceClusterConfig{
```

**Review by bartoszmajsak — COMMENTED** [review]
Just a suggestion - I think `ctrl` suffix might be understood faster by more ppl.

**Review by bartoszmajsak — APPROVED** [review]
LGTM

**Review by ykaliuta — APPROVED** [review]

**Pull request merged**